src/main/kotlin/no/nav/syfo/aktivitetskrav/domain/AktivitetskravStatus.kt

```diff
@@ -28,4 +28,4 @@ private val allowedChangedVurderingStatus = EnumSet.of(
 )
 
 fun AktivitetskravStatus.requiresVurderingArsak(): Boolean =
-    this == AktivitetskravStatus.AVVENT || this == AktivitetskravStatus.UNNTAK || this == AktivitetskravStatus.OPPFYLT
+    this == AktivitetskravStatus.AVVENT || this == AktivitetskravStatus.UNNTAK || this == AktivitetskravStatus.OPPFYLT || this == AktivitetskravStatus.IKKE_AKTUELL
```

Comment:
Må passe på å deploye dette samtidig med frontend-endringen ellers vil det vel feile her når man mangler årsak :boom:

Reply:
Kan og legge inn en toggle for å få en enda smoothere overgang. Eventuelt endre valideringsfunksjonen🤔 Det er jo litt flytende hva vi har årsaker for og ikke. Ser ikke helt verdien i å kaste exceptions backend om det kommer inn en vurdering frontend. Dette styres jo frontend, og testes der. Hva tenker du?

Reply:
Kanskje like greit å vente med valideringens-koden her til vi har det på plass i frontend :thinking:  Tenker jo at det er noe verdi i at domenet i backend sier fra om feil tilstand, selv om man sikkert kunne modellert det så man slapp exceptions.